ansible/templates/telegraf_ports.conf.j2

```diff
@@ -0,0 +1,31 @@
+{% if deploy_host == "blockchain.acuerdo.com" %}
```

Comment:
"blockchain.acuerdo.com" is too specific
and we dont need to put a conditional around the whole block
we can just replace the port part with something like: {{ "18332" if testnet else "8332" }}

Reply:
"blockchain.acuerdo.com" does not work for when we deploy to other domains "bronze.exchange" or when deploying locally (or someone else testing)